### DISCUSSION TITLE
package std:accounts-ui 1.3.1 dependents on email@1.0.10 package

### DISCUSSION THREAD

**Discussion by modanwalrahul31** [meteor/meteor#11733] [Compat Packages]
I am getting the below issue when I upgrade the meteor version from 1.6 to 2.2.3. Please help us with this package to upgrade the meteor version.

 => meteor update --release 2.2.3
=> Errors while initializing project:         
                                              
While selecting package versions:
error: Conflict: Constraint email@1.0.10 is not satisfied by email 2.0.0.
Constraints on package "email":
* email@2.0.0 <- top level
* email@~2.0.0 <- top level
* email@2.0.0 <- accounts-password 1.7.0
* email@1.0.10 <- std:accounts-ui 1.3.1
* email@2.0.0 <- mdg:meteor-apm-agent 3.3.0

**Comment by StorytellerCZ** [comment]
I have created a PR that should fix the issue.
https://github.com/studiointeract/accounts-ui/pull/144
The rest is on the package maintainer. If you want you can either fork it locally with the fix in to get the change immediately.
As for moving the package into compat packages, that is up to @filipenevola

**Comment by filipenevola** [comment]
Hi, @modanwalrahul31 this package already has a maintained fork with the latest email package as a dependency.

Just replace it with this package https://atmospherejs.com/epotek/accounts-ui
